### ISSUE TITLE
Some features needed 

### ISSUE THREAD

**Issue by onos1** [Hede-io/hede.io#23]
While creating a topic 
- adding contributors to that topic should be part of the advanced options, since we have theme, tags, rewards option, contributors also should be included, because some topics involved lot of people to carry out but only one can post it and adding others as contributors will be nice.

- also the opening page should have  view toggle, e.g grid view where uses can see a slide show or images of a particular topic, it will be appeasing to read a topic someone hasnt read when he can see an image about it and the image looks appeasing to open.

**Comment by ercu** [comment]
Thank you for the suggestions!

For the homepage suggestion, it will be very nice to show images with topics. Actually, we would change the homepage with random lists of entries in the future, but your suggestion looks nicer and more appealing.

For "adding contributors" suggestion, anyone can add a new entry to an existing topic. But you ask 1 entry could be written by multiple authors so we tag them? Unlike other wikis editing just a big text area, each author can add their own part to be able to get rewarded on Steem.

**Comment by onos1** [comment]
Yeah, you are right about the contributors part, they can add their contributions manually and get rewarded separately... Wanted to see e.g onos et al  for some contributions, but the existing idea is fine.

Will make a more detailed Steemit post regarding the home page.

**Comment by onos1** [comment]
https://steemit.com/utopian-io/@onos/hede-io-landing-page-view-toggle-ccef79223f981

Here is a full post of the idea, thanks .
